src/OrbitGgp/ProjectTest.cpp

```diff
@@ -82,4 +82,26 @@ TEST(OrbitGgpProject, GetListFromJson) {
   }
 }
 
+TEST(OrbitGgpProject, EqualToOperator) {
+  Project project_0;
+  Project project_1;
+
+  EXPECT_EQ(project_0, project_1);
```

Comment:
Why we are using `EXPECT_EQ(project_0, project_1)` rather than `EXPECT_TRUE(project_0 == project_1)` for the equal case? Will `EXPECT_EQ` call the `==` operator defined in the project struct for comparison?

Reply:
I usually prefer the `EXPECT_EQ` and `EXPECT_NE` macros. They call `operator==` and `operator!=`. So I wanted to use both of them, but since `Project` does not implement `operator!=` I could not use `EXPECT_NE`. Thats why I the the `EXPECT_FALSE(... == ...)`. I hope that makes sense